### PR TITLE
expand templates support

### PR DESCRIPTION
**Pull request by pxue**
@VojtechVitek 
btw are they looking to write actual tests?

### CODE REVIEW

**Comment by VojtechVitek** [comment]
@pxue - @yepher mentioned a desire to refactor tests in #23

File: events/message.go [hunk 1]
```diff
@@ -111,6 +111,10 @@ type Bounce struct {
 	TransmissionID  string            `json:"transmission_id"`
 }
 
+func (b *Bounce) EventType() string {
```
Comment (yepher) [inline events/message.go]: What is the purpose of this. It seems like it is either valuable on all the events or more likely redudnant with 
    `func (e EventCommon) EventType() string { return e.Type }`

Comment (pxue) [inline events/message.go]: good point. EventCommon already implements the interface so this is redundant, will update.

**Comment by yargevad** [comment]
@puxe hanks for the PR! I'm merging this with #54 right now

**Pull request merged**